openprescribing/matrixstore/management/commands/matrixstore_set_live.py

```diff
@@ -0,0 +1,73 @@
+"""
+Updates the live version of the MatrixStore by updating the symlink at
+MATRIXSTORE_LIVE_FILE. If a date is supplied it will use the most recently built file
+with data for that date (which it will search for in MATRIXSTORE_BUILD_DIR). If
+a filename is supplied it will use that file.
+"""
+import os
+import re
+
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from matrixstore.build.common import get_temp_filename
+
+
+class Command(BaseCommand):
+    help = __doc__
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--date',
+            help='Find only files with data up to this date (YYYY-MM format)'
+        )
+        parser.add_argument(
+            '--filename',
+            help="Don't search for files; just use this one"
+        )
+
+    def handle(self, date=None, filename=None, **kwargs):
+        symlink = settings.MATRIXSTORE_LIVE_FILE
+        if os.path.exists(symlink) and not os.path.islink(symlink):
+            raise RuntimeError(
+                'MATRIXSTORE_LIVE_FILE is not a symlink: {}'.format(symlink)
+            )
+        if filename:
+            target_file = get_target_file(filename)
+        else:
+            target_file = get_most_recent_file(date)
+        self.stdout.write(
+            "Updating live symlink to: {}".format(target_file)
+        )
+        temp_file = get_temp_filename(symlink)
+        os.symlink(target_file, temp_file)
+        os.rename(temp_file, symlink)
+        self.stdout.write(
+            "NOTE: You will need to restart the application in order for this "
```

Comment:
I don't know what happens generally to stdout in the pipeline, but this message was specifically for people running the command manually, who'll be most in need of the reminder. I don't think it would serve much use as a line in the pipeline logs.